pkgs/development/python-modules/opensfm/default.nix

```diff
@@ -44,19 +44,15 @@ let
 in
 buildPythonPackage rec {
   pname = "OpenSfM";
-  version = "0.5.2";
+  version = "0.5.2pre";
 
   src = fetchFromGitHub {
     owner = "mapillary";
     repo = pname;
-    rev = "79aa4bdd8bd08dc0cd9e3086d170cedb29ac9760";
-    sha256 = "sha256-dHBrkYwLA1OUxUSoe7DysyeEm9Yy70tIJvAsXivdjrM=";
+    rev = "536b6e1414c8a93f0815dbae85d03749daaa5432";
+    sha256 = "sha256-Nfl20dFF2PKOkIvHbRxu1naU+qhz4whLXJvX5c5Wnwo=";
```

Comment:
```diff
-    sha256 = "sha256-Nfl20dFF2PKOkIvHbRxu1naU+qhz4whLXJvX5c5Wnwo=";
+    sha256 = "Nfl20dFF2PKOkIvHbRxu1naU+qhz4whLXJvX5c5Wnwo=";
```

Reply:
Is this the preferred way? I've been putting like this because this is how nix prints it, and because it's what seemed to be prevalent in the derivations I saw. I'm OK switching if there's a reference I can send the next reviewer to as a justification :)

Reply:
7 bytes shorter = 7 good reasons

the sha256- prefix is needed for `hash = "...."`